pkg/apis/core/validation/validation_test.go

```diff
@@ -20503,3 +20503,59 @@ func TestValidateAppArmorProfileFormat(t *testing.T) {
 		}
 	}
 }
+
+func TestValidatePVSecretReference(t *testing.T) {
+	rootFld := field.NewPath("name")
+	type args struct {
+		secretRef               *core.SecretReference
+		allowDNSSubDomainFormat bool
+		fldPath                 *field.Path
+	}
+	tests := []struct {
+		name        string
+		args        args
+		expectError bool
```

Comment:
Would like to see we verify that the actual error message matches what we expect

Reply:
done